bravado_asyncio/http_client.py

```diff
@@ -142,7 +142,11 @@ def request(
             url=request_params.get('url'),
             params=params,
             data=data,
-            headers=request_params.get('headers'),
+            headers={
+                # Convert not string headers to string
+                k: str(v)
+                for k, v in request_params.get('headers', {}).items()
+            },
```

Comment:
We should do what we do for requests and fido, which is `str(v) if not isinstance(v, bytes) else v`

Reply:
I've checked how requests client and fido are able to translate bytes for headers (as aiohttp client does not support bytes in the headers).

I found that requests depends on urllib3 which uses latin1 encoding, while fido uses yelp_bytes library to perform the encoding (which uses utf-8 or latin1 according to the cases).

I'm using yelp_bytes as the approach is less restrictive and showed no issues with fido